zkevm-circuits/src/evm_circuit.rs

```diff
@@ -310,11 +310,7 @@ pub mod test {
             config
                 .copy_table
                 .load(&mut layouter, &self.block, self.block.randomness)?;
-            config.keccak_table.load(
-                &mut layouter,
-                &self.block.sha3_inputs,
-                self.block.randomness,
-            )?;
```

Comment:
Shouldn't this be changed from `load` to `dev_load` (and not completely removed)?

Reply:
Agree. Let me edit it! :smile: